src/color-picker/color-picker.scss

```diff
@@ -49,6 +49,7 @@ $iui-hover-box-shadow: 0 0 1px $iui-xxs + 1 rgba(var(--iui-color-foreground-body
   margin-right: $iui-xs;
   position: relative;
   background-color: var(--iui-color-swatch-background);
+  forced-color-adjust: none;
```

Comment:
I may have missed this earlier but are there any recommendations to do this? Aren't we breaking anything?

Reply:
And not missing this?
![image](https://user-images.githubusercontent.com/81580355/161515032-af3e3fb5-8fa7-4fec-94c8-8523df090871.png)

Reply:
It only affects high contrast mode, so it doesn't break anything else and media query is not really necessary

https://blogs.windows.com/msedgedev/2020/09/17/styling-for-windows-high-contrast-with-new-standards-for-forced-colors/